rllib/core/models/configs.py

```diff
@@ -107,6 +108,51 @@ def build(self, framework: str = "torch") -> Model:
             return TfMLPHead(self)
 
 
+@ExperimentalAPI
+@dataclass
+class CNNEncoderConfig(ModelConfig):
+    """Configuration for a convolutional network.
+
+    See ModelConfig for usage details.
+
+    Attributes:
+        input_dims: The input dimension of the network. It cannot be None.
```

Comment:
What does input_dims mean for CNN? is it C, H, W? then it should be a tuple of three ints? not just one int? I am confused about this.